regamedll/dlls/player.cpp

```diff
@@ -8013,13 +8048,20 @@ CBaseEntity *EXT_FUNC CBasePlayer::__API_HOOK(DropPlayerItem)(const char *pszIte
 		Vector vecAngles   = pev->angles;
 		Vector vecVelocity = gpGlobals->v_forward * 300 + gpGlobals->v_forward * 100;
 
+#ifdef REGAMEDLL_ADD
+		bool bPackAmmo = (int)ammodrop.value >= 2 ? true : false;
+#else
+		bool bPackAmmo = false;
+#endif
```

Comment:
```suggestion
bool bPackAmmo = false;

#ifdef REGAMEDLL_ADD
    if (ammodrop.value >= 2.0f)
		    bPackAmmo = true;
#else
```

It might be more comfortable to read this way.